apps/portlet-documents/src/main/resources/locale/portal/webui_nl.properties

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright (C) 2014 eXo Platform SAS.
```

Comment:
please update the License Headers : 
1/ AGPL instead of LGPL
2/ 2022 instead of 2014